packages/client/src/components/Location/LocationFinder.test.js

```diff
@@ -137,7 +153,7 @@ describe("LocationFinder", () => {
         target: { value: "666" },
       });
     });
-
-    expect(setErrorMessage).toBeCalled();
+    await waitFor(() => screen.getByTestId(LOCATION_NOT_FOUND));
```

Comment:
You removed the `expect(setErrorMessage).toBeCalled();` line, can you re-add this with your new `setError` function?

Reply:
This issue is not resolved, because this test makes no sense now. I will have a look

Reply:
I've fixed this test, see [commit](https://github.com/Amsterdam/vergunningcheck/pull/685/commits/d999a0cdf94e511c8b3c6ac5f8996c8680dcf533)